.github/workflows/semantic-pr.yml

```diff
@@ -1,18 +0,0 @@
-name: Lint
-
-on:
-  pull_request_target:
-    types:
-      - opened
-      - edited
-      - synchronize
-
-jobs:
-  semantic-pr:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: amannn/action-semantic-pull-request@v5.2.0
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          validateSingleCommit: true
```

Comment:
This requires we have things like `feat:`, `ci:`, `fix:` in our titles for a PR to pass CI.